src/js/helpers/util.js

```diff
@@ -1,3 +1,5 @@
+import regeneratorRuntime from 'regenerator-runtime';
```

Comment:
Hmm, why is this necessary?

Reply:
The tests fail if I don't import regenerator-runtime, I'm not totally sure why though. The failures say that regeneratorRuntime is undefined, then when I import it here the error goes away 🤷‍♀️